api-specs/gem.proto

```diff
@@ -31,14 +31,24 @@ message Gem {
   int64 receiver_id = 7;
   google.protobuf.Timestamp received_at = 8;
   GemColor color = 9;
+  bytes photo = 10;
 }
 
-message DropRequest {
+message GemMessage {
   string message = 1;
+  double latitude = 2;
+  double longitude = 3;
+  int64 creator_id = 4;
```

Comment:
hmm, I think we probably don't need creator_id if we can retrieve the person who sent the request from the backend's context?

Reply:
removed